src/alert/alert.style.ts

```diff
@@ -0,0 +1,205 @@
+import { css } from 'lit';
+// TODO: fix
+
+export default css`
+  :host {
+    box-sizing: border-box;
+  }
+
+  :host *,
+  :host *::before,
+  :host *::after {
+    box-sizing: inherit;
+  }
+
+  [hidden] {
+    display: none !important;
+  }
+
+  :host {
+    display: inline-flex;
+    user-select: none;
+    text-overflow: ellipsis;
+    place-content: center;
+    place-items: center;
+    position: relative;
+  }
+
+  .alert {
+    /* alignment */
```

Comment:
I think comments are not necessary.

Reply:
But aren't they nice to have? I think they can help the readability of the styles